### ISSUE TITLE
Fix aspect Ratio for web

### ISSUE THREAD

**Issue by mannprerak2** [mannprerak2/event_hub_flutter#2]
The app automatically covers up the entire width for web on desktop, This behaviour should be changed and the aspect ratio of a mobile app should be maintained.

Notes
- Only change this for desktop web.
- The changes should not affect android or mobile web

**Comment by jainayu** [comment]
@mannprerak2,  I am still confused about this issue. Can you please explain it a little?

**Comment by mannprerak2** [comment]
Hi @jainayu 
So basically it's meant to target web on desktop.
On desktop this app takes up the entire width, and that looks weird, because this UI was meant to be used via a phone. (where height is much more than the width)

So I think we can fix this (for web on desktop) to have an aspect ratio of a mobile app in potrait mode.

I hope this clears everything

**Comment by jainayu** [comment]
![Screenshot from 2020-10-07 00-42-34](https://user-images.githubusercontent.com/33522478/95249301-1516bd00-0836-11eb-926c-bb88d9dc4001.png)
So do you mean like this?

**Comment by mannprerak2** [comment]
Yes, but width can be further reduced to match the aspect ration of a mobile screen approximately.

**Comment by jainayu** [comment]
Ok. Can you assign this issue to me. I'll finish the rest.
